docker/network/functional/Dockerfile.centos-8

```diff
@@ -1,14 +1,20 @@
-FROM quay.io/ovirt/buildcontainer:stream8
+FROM centos/centos:stream8
 
 # Add runtime dependencies.
 RUN dnf -y install dnf-plugins-core \
+    && \
+    dnf copr enable -y ovirt/ovirt-master-snapshot centos-stream-8 \
+    && \
+    dnf install -y ovirt-release-master \
```

Comment:
In other containers we use different format:

   && do something ... \
      arg \
      arg \
   && next ...

Of course container already use this foramt. If will be nice to unify all container
to use the same format later.